### PR TITLE
Gidk 1kb6qj architecure ui test

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/shared/header/header.component.ts [hunk 1]
```diff
@@ -111,6 +111,7 @@ export class HeaderComponent implements OnInit, AfterViewInit, OnDestroy, AfterV
     @ViewChild('navigationModal', {static: true}) public navigationModal: TemplateRef<any>; // CMD + K
     @ViewChild('dateRangePickerCmp', {static: true}) public dateRangePickerCmp: ElementRef;
     @ViewChild('dropdown', {static: true}) public companyDropdown: BsDropdownDirective;
+    @ViewChild('dropdown', { static: true }) public swithcBranchDropdown: BsDropdownDirective;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/header/header.component.ts:114]: fix spelling of switch

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/shared/header/header.component.ts (hunk 1 above)
Comment (prateek-giddh) [inline apps/web-giddh/src/app/shared/header/header.component.ts:114]: Add JS doc

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/shared/header/header.component.ts (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/shared/header/header.component.ts]: removed code, it is unnecessary code

Comment (shalinee-walkover) [reply apps/web-giddh/src/app/shared/header/header.component.ts]: removed code, it is unnecessary code

**Review by ravinderwalkover — APPROVED** [review]

**Review by prateek-giddh — APPROVED** [review]

**Pull request merged**